app/views/lists/_index.html.erb

```diff
@@ -1,25 +1,35 @@
-<% lists.each do |list| %>
-  <div class="card">
-    <% if user_signed_in? %>
-      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
-    <% end %>
-    <span class="info">
-      <h2><%= list.name %></h2>
-      <p><%= list.description %></p>
-      <p>Tags:<i><%= list.tag_list %></i></p>
-    </span>
-    <span class="content">
-      <%= render partial: 'lists/references', locals: { list: list } %>
-    </span>
-    <span class="options">
-      <ul>
-        <li><%= link_to 'Show', list %></li>
+<% lists.each_slice(3) do |three_lists| %>
```

Comment:
I strongly recommend making whatever changes are necessary so that the HTML
and CSS are the only things affecting the presentation. Can't look at it
right now, but we shouldn't have to do anything like `each_slice`

On Fri, Aug 12, 2016, 2:11 PM wsf246 <notifications@github.com> wrote:

> In app/views/lists/_index.html.erb
> <https://github.com/briankung/projectcredo/pull/52#discussion_r74645286>:
>
> > -  <div class="card">
> > -    <% if user_signed_in? %>
> > -      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
> > -    <% end %>
> > -    <span class="info">
> > -      <h2><%= list.name %></h2>
> > -      <p><%= list.description %></p>
> > -      <p>Tags:<i><%= list.tag_list %></i></p>
> > -    </span>
> > -    <span class="content">
> > -      <%= render partial: 'lists/references', locals: { list: list } %>
> > -    </span>
> > -    <span class="options">
> > -      <ul>
> > -        <li><%= link_to 'Show', list %></li>
> > +<% lists.each_slice(3) do |three_lists| %>
>
> So it looks like there might be something with the alignment if we don't
> make a new row for every 3 cards/panels. With each_slice(3): [image: Inline
> image 1] Without slice: [image: Inline image 2] So i think becomes what we
> decide to do from a formatting perspective. On Fri, Aug 12, 2016 at 2:04
> PM, William Fan <will.shao.fan@gmail.com> wrote:
> … <#m_-485364120015199334_>
> ok ill give it a shot. On Fri, Aug 12, 2016 at 1:59 PM, Brian Kung
> ***@***.***> wrote: > In app/views/lists/_index.html.erb > <#52 (comment)
> <https://github.com/briankung/projectcredo/pull/52#discussion_r74643513>>:
> > > > - <div class="card"> > > - <% if user_signed_in? %> > > - <%= render
> partial: 'votes/toggle_like', locals: { votable: list } %> > > - <% end %>
> > > - <span class="info"> > > - <h2><%= list.name %></h2> > > - <p><%=
> list.description %></p> > > - <p>Tags:<i><%= list.tag_list %></i></p> > > -
> </span> > > - <span class="content"> > > - <%= render partial:
> 'lists/references', locals: { list: list } %> > > - </span> > > - <span
> class="options"> > > - <ul> > > - <li><%= link_to 'Show', list %></li> > >
> +<% lists.each_slice(3) do |three_lists| %> > > If you set the width to 4
> columns or whatever (out of 12) > > — > You are receiving this because you
> authored the thread. > Reply to this email directly, view it on GitHub > <
> https://github.com/briankung/projectcredo/pull/52/files/b653b00e5e4f23efb6cb495404de572981f0e34c#r74643513>,
> > or mute the thread > <
> https://github.com/notifications/unsubscribe-auth/ACKsZKbEgzrSAIMg1wslob-dimdLdPfQks5qfMJ3gaJpZM4JjW-e>
> > . >
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/52/files/b653b00e5e4f23efb6cb495404de572981f0e34c#r74645286>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGx3x9MdMhgo6x8FBKYhlo6Tc-2hsKks5qfMUugaJpZM4JjW-e>
> .
>